bounties/BA122402.md

```diff
@@ -0,0 +1,47 @@
+---
+code: BA122402
+title: Smart Node UI Wireframes
+proposal-author: Patches
+creation-date: 2024-05-28
+external-status: open
+summary: Design wireframes that could be used by a future UI developer and graphic designer to create a front-end for Smart Node.
+reward: 
+  unit: USD
+  max: 2000
+  min: 2000
+repeatable: false
+expiry: July 15, 2024
+application-url: https://dao.rocketpool.net/t/round-12-call-for-bounty-applications-deadline-is-may-7/2919/3?u=shfryn
+skillsets:
+  - node-operation
```

Comment:
Feels like there is room for a 'ux design' skillset? If you want to add one I can add an icon for it later.

Reply:
Added - we'll probably use it a lot.